spec/unleash/scheduled_executor_spec.rb

```diff
@@ -49,4 +49,46 @@
 
     Unleash.configuration.instance_id = original_instance_id
   end
+
+  # These two tests are super flaky because they're checking if threading works
+  # We could extend the times to make them less flaky but that would mean slower tests so I'm disabling them for now
```

Comment:
Tests that include `sleep` are brittle by definition. The tests should be rewritten, and actually we already agreed to re-evaluate `immediate_execution`.

Reply:
Agreed, they're disabled for now, they were useful for local dev but I don't want to patch them if we're going to rewrite that section in a week